webserver/Dockerfile.webserver.dockerignore

```diff
@@ -16,6 +16,5 @@
 !ml_models/
 !database/
 !helpers/
-!config.py
```

Comment:
I don't get where you are getting the config from if you are ignoring it

Reply:
I'm mounting the config as a volume.
If you look in __init__.py there is a check, that looks if a local config should be used or the mounted one.
Like this we can upload our Docker images...

Reply:
But we are using the config in like 4 other files that are included in the webserver as well, so the default config location is still needed.